src/commands.ts

```diff
@@ -64,6 +64,8 @@ export namespace Commands {
 
     export const JAVA_PROJECT_UPDATE = "java.project.update";
 
+    export const JAVA_PROJECT_RELOAD_ALL = "java.project.reloadProjectFromActiveFile";
```

Comment:
`JAVA_PROJECT_RELOAD_ALL` seems to reload all the projects, but it seems to update only the active build file's project in the command execution. Which is right?

Reply:
My bad, should be `JAVA_PROJECT_RELOAD_ACTIVE_FILE`